test_data/context.json

```diff
@@ -176,7 +176,6 @@
         "membership": "join"
       },
       "event_id": "$151396611913abyeC:localhost",
-      "membership": "join",
```

Comment:
Let's not remove this, it seems to be useful if deserialization is borked for additional fields that should be ignored.